pootle/static/js/browser/components/TopContributors.js

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (C) Pootle contributors.
+ *
+ * This file is a part of the Pootle project. It is distributed under the GPL3
+ * or later license. See the LICENSE file for a copy of the license and the
+ * AUTHORS file for copyright and authorship information.
+ */
+
+import React from 'react';
+
+import Avatar from 'components/Avatar';
+
+
+function t(string, args) {
```

Comment:
Some background information on this one: I shared a quick snippet via IM with @ta2-1 to perform simple interpolating instead of relying on the heavy function at _utils/i18n_. I don't believe there was any extra thought put on its location, and as you say _utils/i18n_ sounds good.

There's the catch of having already a `t()` function there though. I suggest we rename that function, which accepts components as arguments, to something like `tct()` and have both of them in _utils/i18n_.